src/components/approvallist/ApprovalEntry.tsx

```diff
@@ -24,6 +27,7 @@ export const ApprovalEntry: React.FC<ApprovalEntryProps> = observer((props) => {
       <AccordionSummary>
         <ColumnGrid>
           <FlexRowWrapper>
+            {/* It's very picky, but I would stop the propagation where the event occurs: before calling `toggleSelected`. */}
```

Comment:
@iamacook 
I fixed most of the review issues. But this one doesn't work.
I think because the safe react components Checkbox only exposes the onChange event handler and not the onClick handler. So the Accordion will always toggle when the user clicks on the checkbox.